client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -171,12 +171,11 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
           onChange(newNum);
         }}
         onBlur={() => {
-          if (isDirty) {
```

Comment:
We shouldn't lose this `isDirty` check

Reply:
good catch! thanks - there was a conflict in this component, though not at this point. this one is a plain GH merge thing..

Reply:
Ah yeah. Two PRs (#3884 & #3825) both targeting the same area in the same file, one in develop and the other in the release branch. Dangerous ground!

<img width="646" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/23d54bcf-e3c6-4877-a87a-4d54450b86c2">